hips/utils/wcs.py

```diff
@@ -4,16 +4,20 @@
 import numpy as np
 from astropy.coordinates import SkyCoord
 from astropy.wcs import WCS
+from astropy.coordinates import Angle
 from astropy.wcs.utils import pixel_to_skycoord, wcs_to_celestial_frame
 
 __all__ = [
     'WCSGeometry',
 ]
 
-__doctest_skip__ = ['WCSGeometry']
+__doctest_skip__ = [
+    'WCSGeometry',
+    'WCSGeometry.*'
```

Comment:
I find that weird that listing a class doesn't mean skip all doctests (including methods) in that class. But OK, if that's how it works, then keep as is of course.